metrics/resource_view.go

```diff
@@ -295,6 +287,21 @@ func resourceFromKey(s string) *resource.Resource {
 	return r
 }
 
+func copyViews(views []*view.View) []*view.View {
+	viewsCopy := make([]*view.View, 0, len(resourceViews.views))
+	for _, v := range views {
+		c := *v
+		c.TagKeys = make([]tag.Key, 0, len(v.TagKeys))
+		c.TagKeys = append(c.TagKeys, v.TagKeys...)
+		c.Aggregation = new(view.Aggregation)
+		*c.Aggregation = *v.Aggregation
```

Comment:
```suggestion
agg := *v.Aggregation
c.Aggregation = &agg
```

Reply:
Isn't this equivalent?

Reply:
Probably, but we're mostly doing this in the codebase. I.e. readability mostly.